src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -187,6 +187,35 @@ public function itGivesFixedOffEveryProductClassifiedAs(
     }
 
     /**
+     * @Given /^([^"]+) gives ("(?:€|£|\$)[^"]+") off on every product (more|less) expensive than ("(?:€|£|\$)[^"]+")$/
+     */
+    public function thisPromotionGivesOffOnEveryProductMoreLessExpensiveThan(
+        PromotionInterface $promotion,
+        $discount,
+        $limit,
+        $amount
+    ) {
+        $limitType = ('more' === $limit) ? 'min' : 'max';
+        $filterConfiguration = ['filters' => ['price_range' => [$limitType => $amount]]];
+
+        $this->createPromotionWithPriceRangeFilter($promotion, $discount, $filterConfiguration);
+    }
+
+    /**
+     * @Given /^(this promotion) gives ("(?:€|£|\$)[^"]+") off on every product more expensive than ("(?:€|£|\$)[^"]+") and less expensive than ("(?:€|£|\$)[^"]+")$/
+     */
+    public function thisPromotionGivesOffOnEveryProductMoreExpensiveThanAndLessExpensiveThan(
+        PromotionInterface $promotion,
+        $discount,
+        $minAmount,
+        $maxAmount
+    ) {
+        $filterConfiguration = ['filters' => ['price_range' => ['min' => $minAmount, 'max' => $maxAmount]]];
+
+        $this->createPromotionWithPriceRangeFilter($promotion, $discount, $filterConfiguration);
+    }
```

Comment:
@Zales0123 Why? We spec all contexts... Do not add them now, because it is waste to spec existing code. Write specs as soon as you change something here.